components/badges/badges.html

```diff
@@ -0,0 +1,118 @@
+                        <li>Zingy UI</li>
+                    </a>
+                    <a href="/components/avatar/avatar.html">
+                        <li>Docs</li>
+                    </a>
+                </div>
+                <div class="nav-right flex-evenly">
+                    <a href="https://github.com/praveenreddy1798"><i class="fa fa-github fa-2x"
+                            aria-hidden="true"></i></a>
+                    <a href="https://twitter.com/praveen1798"><i class="fa fa-twitter fa-2x" aria-hidden="true"></i></a>
+                    <a href="https://www.linkedin.com/in/praveen-kumar-d-06b545168/"><i class="fa fa-linkedin fa-2x"
+                            aria-hidden="true"></i></a>|
+                </div>
+                <div class="mobile-menu flex-evenly align-center">
+                    <button class="btn hamburger mobile-item bg-primary white-color"><i
+                            class="fa fa-bars fa-2x h-100"></i></button>
+                    <button class="btn hamburger-close mobile-item bg-primary white-color"><i
+                            class="fa fa-close fa-2x h-100"></i></button>
+                </div>
+            </ul>
+        </nav>
+
+        <!-- Sidebar -->
+        <div class="sidebar">
+            <ul class="sidebar-items flex-vertical">
+                <a href="/components/avatar/avatar.html">
+                    <li>Avatar</li>
+                </a>
+                <a href="/components/alerts/alerts.html">
+                    <li>Alerts</li>
+                </a>
+                <a class="active-sidebar-item" href="/components/badges/badges.html">
+                    <li>Badges</li>
+                </a>
+                <a href="/components/buttons/buttons.html">
+                    <li>Buttons</li>
+                </a>
+                <a href="/components/cards/cards.html">
+                    <li>Cards</li>
+                </a>
+                <a href="/components/responsive-images/responsive-images.html">
+                    <li>Responsive Images</li>
+                </a>
+                <a href="/components/input/input.html">
+                    <li>Input</li>
+                </a>
+            </ul>
+        </div>
+
+        <!-- Badges -->
+        <main class="main-section mg-md">
+            <h1>Badges</h1>
+            <h3 class="mg-t-xsm">
+                Avatar Badge
+            </h3>
+            <div class="flex-evenly mg-xlg">
+                <!-- Avatar Badge -->
+                <div class="badge-container avatar-badge">
+                    <img src="../../assets/profile.jpg" alt="avatar" class="avatar avatar-l" />
+                    <span class="rounded">1</span>
+                </div>
+            </div>
+            <div class="flex-evenly mg-t-lg">
+                <iframe src="https://carbon.now.sh/embed/a7ywQ74MK3MjdCbhkoxW"
+                    style="width: 640px; height: 200px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
1. can increase height so that vertical scroll will be removed from UI
2. can change to `width:100%`